### PR TITLE
[json/sv] Added Swedish translation for JSON.

### PR DESCRIPTION
**Pull request by larkov**

### CODE REVIEW

File: sv-se/json-sv.html.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "nycklar": "måste alltid omslutas med dubbla citationstecken",
+  "nummer": 0,
+  "strängar": "Alla unicode-tecken (inklusive \"escaping\") är tillåtna.",
+  "boolska värden?": true,
+  "nullvärden": null,
+
+  "stora tal": 1.2e+100,
+
+  "objekt": {
+    "kommentar": "Det flesta datastukturerna i JSON kommer i form av objekt.",
```
Comment (Oire) [inline sv-se/json-sv.html.markdown]: `De flesta datastrukturerna`? Sorry, my Swedish is not so good so far :)

File: sv-se/json-sv.html.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+language: json
+filename: learnjson.json
```
Comment (Oire) [inline sv-se/json-sv.html.markdown]: Skulle vara `learnjson-sv.json`

**Comment by Oire** [comment]
@larkov, thanks a lot! Please see my comments and I'll merge (the first translation into Swedish, btw!)

**Comment by larkov** [comment]
Your edits are correct and your Swedish is excellent  :)

**Comment by Oire** [comment]
@larkov, thanks for your appreciation! Please fix those and I'll merge.

**Comment by Oire** [comment]
Tack så mycket!

**Pull request merged**